package.json

```diff
@@ -15,7 +15,8 @@
     "lint:fix": "eslint . --fix",
     "test-server": "cross-env NODE_ENV=test nodemon server.js",
     "test": "jest --coverage --detectOpenHandles ",
-    "generate": "graphql-codegen"
+    "generate": "graphql-codegen",
+    "ci": "start-server-and-test test-server http://localhost:6000 test"
```

Comment:
Please, rename the script to the `test-full` and take port number from the environment variable

Reply:
fixed